faucet/valve.py

```diff
@@ -924,6 +924,10 @@ def port_delete(self, port_num):
 
     def _reset_lacp_status(self, port):
         self._set_var('port_lacp_status', port.dyn_lacp_up, labels=self.dp.port_labels(port.number))
+        self._notify(
+                {'LAG_CHANGE': {
+                    'port_no': port.number,
```

Comment:
likely also need/want switch name.  E.g. in our setup it will always be port 28... not all that useful :-)

Reply:
We get switch name from the event itself. Same as 'PORT_CHANGE'